_includes/assets/sass/_variables.scss

```diff
@@ -40,26 +40,20 @@ $base-font-size: 19px;
 $h1-font-size-mobile: 1.158rem;
 $h1-font-size: 1.526rem;
 
-$h2-font-size-mobile: 0rem;
-$h2-font-size: 0rem;
+$h2-font-size-mobile: 1rem;
+$h2-font-size: 1.211rem;
```

Comment:
@SamMed1 What do you think about reusing the px-to-rem ultility from govuk frontend to make this easy to read? https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/tools/_px-to-rem.scss

Reply:
@lewisnyman - Apologies for the delayed response. I wanted to test this before giving my thoughts. I think this is a great idea, and will go a long way in making values more readable.

I would like to roll this out across the whole site. As all values so far have been defined in rem, I suggest we merge this branch in, and then create a new branch for implementing this, if that works with you?

Reply:
@lewisnyman - I've just seen that you are on holiday today. My strategy is going to be merging this in, then rolling out the above on a separate branch, just so we can keep a clear record of changes.